README.adoc

```diff
@@ -92,7 +92,7 @@ Now you are ready to https://bitcoinj.github.io/getting-started[follow the tutor
 
 Building apps with official releases of *bitcoinj* is covered in the https://bitcoinj.github.io/getting-started[tutorial].
 
-If you want to develop or test your app with a https://jitpack.io[Jitpack]-powered build of the latest `master` or `release-0.15` branch of *bitcoinj* follow the dynamically-generated instructions for that branch by following the correct link.
+If you want to develop or test your app with a https://jitpack.io[Jitpack]-powered build of the latest `master` or `v0.16.2` branch of *bitcoinj* follow the dynamically-generated instructions for that branch by following the correct link.
```

Comment:
You should use the `release-0.16` _branch_, not the `v0.16.2` _tag_ because that branch will always have the latest code from that branch. There is little value in using Jitpack to reference the 0.16.2 tag because that will always be the same as the 0.16.2 official release.

At the current time the branch and the tag are the same, but if we fix a bug on that branch or start work on a 0.16.3 release that branch and the tag will differ.